### ISSUE TITLE
Unable to run Spek tests from Intellij

### ISSUE THREAD

**Issue by filipproch** [JetBrains/spek#561]
Hi,

I am having an issue with running Spek tests in my Android project from Intellij. If I run them from gradle (`./gradlew test`) it works. Running from intellij, ends up like on picture bellow. No error, just no tests being run :(

<img width="1073" alt="screenshot 2019-01-10 at 12 02 52 am" src="https://user-images.githubusercontent.com/1387116/50931487-60d2e180-146b-11e9-9839-e2101f8f7a26.png">

Spek Plugin: v2.0.0-rc.1.167+0914c41-IJ2018.3
Kotlin: 1.3.11
Spek: 2.0.0-rc.1
Android JUnit5: 1.3.2.0
Android Gradle Plugin: 3.2.1

I added
```
sourceSets.each {
        it.java.srcDirs += "src/$it.name/kotlin"
    }

    testOptions {
        junitPlatform {
            filters {
                engines {
                    include 'spek2'
                }
            }
            jacocoOptions {
                // here goes all jacoco config, for example
                html.enabled = true
                xml.enabled = false
                csv.enabled = false
                unitTests.all {
                    testLogging.events = ["passed", "skipped", "failed"]
                }
            }
        }
    }
```

and 

```
    // spek
    testImplementation "org.spekframework.spek2:spek-dsl-jvm:$spekVersion"
    testImplementation "org.spekframework.spek2:spek-runner-junit5:$spekVersion"

    // spek requires kotlin-reflect, omit when already in classpath
    testImplementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
```

to my build.gradle (as per tutorial). As I mention, it's working, no problem, from Gradle, but not from IntelliJ.

**Comment by raniejade** [comment]
Running tests for an android project within IntelliJ is unsupported, please use Android Studio.

**Comment by pardom** [comment]
Could you provide some context as to why it is unsupported? IntelliJ uses the same Android plugin that Android Studio does. If you could provide more info, I'm sure the community (myself included) would be happy to contribute.

**Comment by filipproch** [comment]
@raniejade Or at least maybe to write it somewhere? I spent a lot of hours researching why it doesn't work, and all was wasted :/ 
Otherwise, agree with @pardom , happy to help make it work in IntelliJ

**Comment by raniejade** [comment]
@filipproch Yep, should be fixed by #562.

Originally when I wrote the plugin(https://github.com/raniejade/spek-idea-plugin), we only published a single version for both IntelliJ and Android Studio. The AS support was riddled with bugs (special mention: https://github.com/raniejade/spek-idea-plugin/issues/6) as the plugin didn't handle android projects properly. I was able to fix it but it resulted to having to maintain different versions of the plugin. This was due to various tricks I needed to introduce so I can "properly" integrate with the Android plugin. See https://github.com/spekframework/spek/blob/2.x/spek-ide-plugin/android-studio/src/main/kotlin/org/spekframework/intellij/SpekAndroidParameterPatcher.kt#L18, https://github.com/spekframework/spek/blob/2.x/spek-ide-plugin/android-studio/src/main/kotlin/org/spekframework/intellij/SpekAndroidRunConfiguration.kt#L16, https://github.com/spekframework/spek/blob/2.x/spek-ide-plugin/android-studio/src/main/kotlin/org/spekframework/intellij/SpekAndroidConfigurationType.kt#L7. If you check https://github.com/spekframework/spek/tree/2.x/spek-ide-plugin you would see a lot of duplicated code, fixing that requires non-android users of Spek to have the Android plugin installed (or enabled) when using IntelliJ.
